proto/frequenz/api/electricity_trading/v1/electricity_trading.proto

```diff
@@ -583,7 +583,7 @@ message UpdateGridpoolOrderRequest {
 
     // Optional; Updated execution options such as All or None,
     // Fill or Kill, etc.
-    OrderExecutionOption execution_option = 7;
+    optional OrderExecutionOption execution_option = 7;
```

Comment:
No, we had to remove the optional keywords for the types that do not have a 'sane default'. `OrderExecutionOption` should still have an optional keyword because its 'sane default' value is `UNSPECIFIED`. [This](https://github.com/frequenz-floss/frequenz-api-electricity-trading/issues/19) issue has more information about it

Reply:
You are right. I misread and thought this is a message rather then an enum. Sorry for the fuss ;).